l10n_es_aeat_sii_match/models/aeat_sii_match_difference.py

```diff
@@ -7,13 +7,12 @@
 
 class SiiMatchDifferences(models.Model):
     _name = "l10n.es.aeat.sii.match.difference"
+    _description = "SII match difference"
 
-    invoice_id = fields.Many2one(
-        string="Related invoice", comodel_name="account.invoice"
-    )
+    invoice_id = fields.Many2one(string="Related invoice", comodel_name="account.move")
```

Comment:
(commit mig 13.0 https://github.com/OCA/l10n-spain/pull/3151/commits/268165426e207987b72e4cdf2a65f9292507543c), haría falta una migración de datos para los ids de este campo no? Mi compañero está teniendo problemas con eso a la hora de migrar a la 16.0 en una bbdd migrada de <13.0
Como habéis hecho unos commits de migración a la 13.0 por separado pero no está el módulo fusionado en la rama 13.0, creo que lo mejor sería hacer cherry-pick de esos commits de migración y dejar el módulo en la 13.0 junto con los scripts de migración para este campo

@pedrobaeza @victoralmau  ¿Cómo lo veis?

Reply:
Sí, nosotros no tenemos esa necesidad, pero si tienes clientes <=v12, adelante con la propuesta de ese PR.

Reply:
@pedrobaeza He añadido este PR añadiendo el script de migración en la versión 13.0.
https://github.com/OCA/l10n-spain/pull/3249
En tener el PR de la 16.0 tendrá ya  el commit separado que he puesto para el script